ui-ngx/src/app/modules/home/pages/dashboard/dashboard-form.component.ts

```diff
@@ -84,6 +86,7 @@ export class DashboardFormComponent extends EntityComponent<Dashboard> {
         image: [entity ? entity.image : null],
         mobileHide: [entity ? entity.mobileHide : false],
         mobileOrder: [entity ? entity.mobileOrder : null, [Validators.pattern(/^-?[0-9]+$/)]],
+        assignedCustomerIds: [[]],
```

Comment:
It is better to add this control to the form only in cases where a dashboard is created. Example: ota-update.component